.github/workflows/mainnet.yml

```diff
@@ -0,0 +1,38 @@
+name: Mdw Frontend Mainnet
+
+on:
+  push:
+    branches: 
+      - master
+    tags:
+      - 'v*'
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+  
+    name: Build Docker
+    steps:
+
+      - uses: actions/checkout@v2
+  
+      - name: Log in to Docker Hub
+        if: github.event_name != 'pull_request'
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USER }}
+          password: ${{ secrets.DOCKERHUB_PASS }}
+
+      - name: Build and push Docker image
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          file: nginx.Dockerfile
+          build-args: |
+            NUXT_APP_NODE_URL: "https://mainnet.aeternity.io/mdw"
+            NUXT_APP_NODE_WS: "wss://mainnet.aeternity.io/mdw/websocket"
+            NUXT_APP_NETWORK_NAME: "MAIN NET"
+            NUXT_APP_ENABLE_FAUCET: "false"
+            NUXT_APP_FAUCET_API: "https://testnet.faucet.aepps.com/account"
+          push: ${{ github.event_name != 'pull_request' }}
+          tags: aeternity/mdw-frontend:mainnet
```

Comment:
IMHO this should be `aeternity/mdw-frontend-tesnet:latest` or something like that

Reply:
We discussed it with Dincho I think , we have to sync on that